tests/fixtures/config-hierarchy/file-structure.json

```diff
@@ -48,6 +48,12 @@
             ".eslintrc": "{\n    \"env\": {\n        \"commonjs\": true\n    }\n}\n"
         }
     },
+    "overrides": {
```

Comment:
Just double-checking: are you sure this won't interfere with already-existing tests?